app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -372,16 +372,10 @@ def parse_asset_details(node)
     end
 
     def find_loc_led_state(leds)
-      loc_led_state = ""
-      unless leds.nil?
-        leds.each do |led|
-          if led["name"] == "Identify"
-            loc_led_state = led["state"]
-            break
-          end
-        end
-      end
-      loc_led_state
+      identification_led_name = %w(Identify Identification)
+      identification_led = leds&.find { |led| identification_led_name.include?(led["name"]) }
+      return identification_led["state"] unless identification_led.nil?
+      ""
```

Comment:
Does this _have_ to be `""` or is `nil` okay?
How about just `identification_led.try(:[], "state")`